lib/plugins/features/movies/index.js

```diff
@@ -38,14 +38,14 @@ exports.register = (server, options, next) => {
       path: '/movies/{id}/locations',
       config: {
         handler: (request, reply) => {
-          reply(Controller.addLocationToMovie(request.params.id, request.payload.location_id));
+          reply(Controller.addLocationToMovie(request.params.id, request.payload.location_name));
```

Comment:
nit: pass in just the `request.payload` to the controller method. this is useful for when we have a request with a payload containing many attributes.